models/intermediate/int_xero__invoice_line_item_pivoted_tracking_categories.sql

```diff
@@ -0,0 +1,58 @@
+{{ config(enabled=(var('xero__using_invoice_line_item_tracking_category', True)
+        and var('xero__using_tracking_categories', True))) }}
+
+with invoice_line_item_has_tracking as (
+
+    select *
+    from {{ var('invoice_line_item_has_tracking_category') }}
+
+), tracking_category as (
+
+    select *
+    from {{ var('tracking_category') }}
+    where lower(status) in ('active', 'archived')
+
+), tracking_category_has_option as (
+
+    select *
+    from {{ var('tracking_category_has_option') }}
+
+), tracking_category_option as (
+
+    select *
+    from {{ var('tracking_category_option') }}
+    where lower(status) = 'active'
```

Comment:
I noticed we're not using these ctes, but I believe we should be in order to obtain the accurate tracking categories. Is there a reason these are included and not used?

Reply:
@fivetran-joemarkiewicz  @fivetran-jamie and I synced on this during our alignment meeting and were seeing unusual issues with the initial join, so I simplified it to a base format. But probably best to standardize as discussed elsewhere.